src/core/Akka.MultiNodeTestRunner/Akka.MultiNodeTestRunner.csproj

```diff
@@ -73,14 +73,6 @@
       <Project>{964F0EC5-FBE6-47C5-8AE6-145114D5DB8C}</Project>
       <Name>Akka.MultiNodeTestRunner.Shared</Name>
     </ProjectReference>
-    <ProjectReference Include="..\Akka.NodeTestRunner\Akka.NodeTestRunner.csproj">
```

Comment:
I have deleted this by accident when was cleaning now unnecessary Akka.Remote.TestKit